cli/commands.go

```diff
@@ -58,6 +49,14 @@ var (
 				flFormat,
 			},
 		},
+		{
```

Comment:
please define the command like this: https://github.com/bitrise-io/bitrise/blob/master/cli/workflow_list.go#L16
and add it to the command list like this: https://github.com/bitrise-io/bitrise/blob/master/cli/commands.go#L187

Reply:
also do not wire in the flags as a global var (we use it only in the update command's scope)